azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/CallingService.kt

```diff
@@ -87,6 +91,14 @@ internal class CallingService(
         return callingSdk.turnOnMicAsync()
     }
 
+    fun startAudio() {
```

Comment:
Naming, as mentioned elsewhere, it's a bit generic and could use some details. A developer may think of audio within a certain framing, i.e. mic or speaker, but not always both. So useful to specify.

i.e. startIncomingAndOutgoingAudio() {
}

Reply:
As per discussion we will fix for GA.